tests/test_config.py

```diff
@@ -72,6 +72,17 @@ def test_load_from_missing_envvar(app):
         app.config.from_envvar('non-existent variable')
 
 
+def test_load_from_envvar_invalid_syntax(app):
+    config = b"VALUE = some value"
+    with NamedTemporaryFile() as config_file:
+        config_file.write(config)
+        config_file.seek(0)
+        environ['APP_CONFIG'] = config_file.name
+
+        with pytest.raises(SyntaxError):
+            app.config.from_envvar('APP_CONFIG')
```

Comment:
why do from_envvar here? your change is more directly relevant to from_pyfile

Reply:
Sorry I missed it. I replace it with `from_pyfile`